app/views/admin/products_v3/_variant_row.html.haml

```diff
@@ -48,6 +48,7 @@
       selected_option: variant.primary_taxon_id,
       aria_label: t('.category_field_name'),
       placeholder_value: t('admin.products_v3.filters.search_for_categories')))
+  = error_message_on variant, :primary_taxon_id
```

Comment:
We don't have a spec on this, do we?

Reply:
🙈  You're right, I need to add that.

Reply:
Wait, for `primary_taxon_id`, I don't think there is a way to trigger an error. 
I put this in for future-proofing, which I realise is generally a bad idea, but it seemed likely that if someone added a validation condition, they wouldn't think to add this.
It's important on this screen, because we don't have an error summary at the top of the screen.

Reply:
Given that it's untested, there's a very real possibility it doesn't work (eg, any validations would be on the association, not the id!).
So I will remove this.